src/view/tables/CaseDetails.scss

```diff
@@ -7,23 +7,17 @@
   border-right: 1px solid transparent;
 }
 
-.grid-container {
-  margin-left: 0;
-  padding-left: 0;
-  margin-right: 0;
-  padding-right: 0;
-  max-width: 100%;
-}
-
-.case-detail-date {
-  min-width: 11em;
-  font-weight: bold;
-}
-
 .case-detail-list {
-  margin-bottom: 1em;
```

Comment:
Losing this appears to remove some whitespace between the `case-detail-list` and the bottom of the row that I liked (it's the same as the `h3` element padding at the top of the row).